src/Sylius/Bundle/ChannelBundle/Resources/config/services.xml

```diff
@@ -40,6 +40,12 @@
         <service id="sylius.context.channel.composite" class="Sylius\Component\Channel\Context\CompositeChannelContext" public="false" />
         <service id="Sylius\Component\Channel\Context\ChannelContextInterface" alias="sylius.context.channel" />
 
+        <service id="sylius.context.channel.env_based" class="Sylius\Component\Channel\Context\EnvBased\ChannelContext" public="false">
+            <argument type="service" id="sylius.repository.channel" />
+            <argument>%sylius_channel_env%</argument>
```

Comment:
```suggestion
            <argument>%env(SYLIUS_CHANNEL)%</argument>
```

Reply:
I used this way instead of the parameter.